.github/workflows/build.yml

```diff
@@ -18,6 +18,12 @@ jobs:
         - nixosConfigurations.attest-testing.config.system.build.toplevel
         - nixosConfigurations.attest.config.system.build.toplevel
 
+        # Steward production
+        - nixosConfigurations.attest-production-prod1.config.system.build.toplevel
```

Comment:
Why not just
```
nixosConfigurations.attest-1.config.system.build.toplevel
```
?

For example `attest-testing` could then also have `attest-testing-{1,2...,n}`

This would be easier to read and to type (`deploy .#attest-1` etc.)

It does not seem like this fits in with the rest of host names.
We could also revisit the naming scheme if you prefer

Reply:
Because I want to be clear in the system name that it's production: that should be an explicit phrase.
I really don't like the "there's no -testing or -staging, thus it's production", since you may accidentally overlook the suffix.

Additionally, this makes it clear that these are boxes running customer-facing services, rather than e.g. lab machines.